.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+# What
```

Comment:
This seems a bit opinionated. At least, this isn't how I prefer to write PRs. Could we make this change separately?